docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -258,6 +258,12 @@ The `storage` block defined in the `values.yaml` file is used to configure the s
 The procedure below configures MinIO as the local storage option managed by the Helm chart.
 However, you can use a other storage provides. Refer to the Optional storage section below.
 
+{{% admonition type="note" %}}
+
+The MinIO installation that is included with this Helm chart is for demonstration purposes only, and is configured for a maximum storage size of 5GB. This MinIO installation is not suitable for production environments and should only be used for example purposes. Performant, Enterprise-grade object storage should always be used in production
```

Comment:
```suggestion
The MinIO installation that is included with this Helm chart is for demonstration purposes only, and is configured for a maximum storage size of 5GiB. This MinIO installation is not suitable for production environments and should only be used for example purposes. Performant, Enterprise-grade object storage should always be used in production
```
Is this `GB` or `GiB`?

Reply:
@hedss Which one did you want this to be?

Reply:
It should be Gi or GiB.